onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3990,7 +3990,7 @@ def aten_native_group_norm_backward(
 @torch_op("aten::native_layer_norm", trace_only=True)
 def aten_native_layer_norm(
     input: TReal,
-    normalized_shape: Sequence[int],
+    normalized_shape: Sequence[INT64],
```

Comment:
Same question here. Is this supposed to be an int64 tensor, or a sequence of int64 tensors? I doubt it is meant to be the second. Is there a pointer to the corresponding aten op documentation?

Reply:
yes, should be int64 tensor. fixed.